torch_frame/data/mapper.py

```diff
@@ -165,6 +165,47 @@ def backward(self, tensor: MultiNestedTensor) -> pd.Series:
         return pd.Series(ser)
 
 
+class SequenceTensorMapper(TensorMapper):
+    r"""Maps any sequence series into an :obj:`MultiNestedTensor`.
+
+    Args:
+        sep (str): The delimiter for the sequence in each cell.
+        (default: :obj:`,`)
+    """
+    def __init__(self, ):
+        super().__init__()
+
+    def forward(
+        self,
+        ser: Series,
+        *,
+        device: Optional[torch.device] = None,
+    ) -> MultiNestedTensor:
+        values = []
+        num_rows = len(ser)
+        offset = ser.apply(lambda row: len(row) if row is not None and not (
+            isinstance(row, float) and pd.isna(row)) else 0)
```

Comment:
I think it is safer to check if each element of `ser` is a list or not.

Reply:
Fixed